conbench/tests/app/_asserts.py

```diff
@@ -52,7 +52,8 @@ def assert_404_not_found(self, r):
         # TODO: How to make this an HTML error page?
         # While still having json error rs for the API.
         assert r.content_type == "application/json", r.content_type
-        assert r.json == {"code": 404, "name": "Not Found"}, r.json
+        assert r.json["code"] == 404, "name"
```

Comment:
I think "name" isn't intentional here, right?

Reply:
Haha, no. I wonder how it got there, and what the facial expression would have been to discover this later.